From 6426009b9442f9160499563d456ff5d693505b8a Mon Sep 17 00:00:00 2001
From: Inkamari Harjula <inkamari.harjula@qt.io>
Date: Mon, 6 Oct 2025 08:27:56 +0300
Subject: [PATCH 120/239] Doc: Add Qt Academy course links to documentation

Three course links added (QML Dashboard Calendar, Inbox, and Main UI].

Task-number: QTBUG-140851
Change-Id: I1013fa3e0280a6b632826637fd37a83def96e490
Reviewed-by: Alexei Cazacov <alexei.cazacov@qt.io>
(cherry picked from commit aed22a7f668b783c48faa48d5896ac66bbf6fb05)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/quick/doc/src/getting-started/getting-started.qdoc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/quick/doc/src/getting-started/getting-started.qdoc b/src/quick/doc/src/getting-started/getting-started.qdoc
index 6e6ac2c6f3..fd7d0f47da 100644
--- a/src/quick/doc/src/getting-started/getting-started.qdoc
+++ b/src/quick/doc/src/getting-started/getting-started.qdoc
@@ -28,6 +28,9 @@ and Qt Quick:
 \li \l {https://www.qt.io/academy/course-catalog#introduction-to-qt-quick}{Introduction to Qt Quick}
 \li \l {https://www.qt.io/academy/course-catalog#introduction-to-qt-quick-controls}{Introduction to Qt Quick Controls}
 \li \l {https://www.qt.io/academy/course-catalog#creating-a-simple-qt-quick-application-}{Creating a Simple Qt Quick Application}
+\li \l {https://www.qt.io/academy/course-catalog#qml-dashboard:-main-ui}{QML Dashboard: Main UI}
+\li \l {https://www.qt.io/academy/course-catalog#qml-dashboard:-inbox}{QML Dashboard: Inbox}
+\li \l {https://www.qt.io/academy/course-catalog#qml-dashboard:-calendar}{QML Dashboard: Calendar}
 \endlist
 
 \section1 What is QML?
-- 
2.51.2

